server/controller.js

```diff
@@ -1,27 +1,28 @@
-// controllers for requests to server
+// controllers for requests to api routes
+// import mongoose
 const mongoose = require('mongoose');
-const recommendations = require('../database/index.js');
+// const recommendations = require('../database/index');
 
-mongoose.connect('mongodb://localhost/recommendatinos');
+// create connection to database
+mongoose.connect('mongodb://localhost/recommendations', { useNewUrlParser: true });
```

Comment:
It might make more sense to keep opening/closing of the connection in the db file